index.html

```diff
@@ -217,8 +217,6 @@ <h1 id="ovningar">Övningar</h1>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
     <script src="vendor/highlight.pack.js"></script>
     <script src="vendor/bootstrap.min.js"></script>
-    <script src="vendor/smooth-scroll.js"></script>
```

Comment:
> The gulpfile is a common workflow for building Sass etc.
> All in all, a quite common Node stack. npm install and off you go.

I've worked with Gulp, Grunt, node, sass, scss, etc before. Don't be patronizing in your formatting.